src/Graph12/Graph12.tsx

```diff
@@ -0,0 +1,53 @@
+import { ProgressCircle } from "../components/ProgressCircle";
+const data = [
```

Comment:
Please use uppercase for static values :)

Reply:
done